workLoggingSystem.html

```diff
@@ -304,7 +304,7 @@
                         </div>
                         <div class="col">
                             <div class="row">
-                                <div class="col-md-4 menu">Status:</div>
+                                <div class="col-md-4 menu" id="sss">Status:</div>
```

Comment:
No need to give them id, You can directly change the text color of input tag id's to red, will reduce the complexity of the code